### DISCUSSION TITLE
Training error

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#651] [General Q&A]
![image](https://user-images.githubusercontent.com/61519589/125247669-cf9d8980-e2b8-11eb-9cfb-762320073e62.png)

MY CONFIG JSON
`{
    "run_name": "ljspeech",
    "run_description": "finetune 4241 for align with architectural changes",

    "datasets": [
        {
            "name": "ljspeech",
            "path": "/Users/Gustavo/Desktop/ArchivosEntrenados/cuentos_clasicos_del_norte",
            "meta_file_train": "metadata.csv",
            "meta_file_val": null
        }
    ],
    "audio":{
        
        "num_mels": 80,          
        "num_freq": 1025,       
        "sample_rate": 22050,
        "resample":true,   
        "frame_length_ms": 50,  
        "frame_shift_ms": 12.5, 
        "preemphasis": 0.98,    
        "min_level_db": -100,   
        "ref_level_db": 20,     
        "power": 1.5,           
        "griffin_lim_iters": 60,
        "signal_norm": true,    
        "symmetric_norm": false, 
        "max_norm": 1,          
        "clip_norm": true,      
        "mel_fmin": 0.0,         
        "mel_fmax": 8000.0,        
        "do_trim_silence": true  
    },

    "distributed":{
        "backend": "nccl",
        "url": "tcp:\/\/localhost:54321"
    },

    

    "model": "Tacotron2",   
    "grad_clip": 1,      
    "epochs": 1000,         
    "lr": 0.0001,            
    "lr_decay": false,      
    "warmup_steps": 4000,   
    "windowing": false,      
    "memory_size": 5,       
    "attention_norm": "softmax",   
    "prenet_type": "bn",    
    "use_forward_attn": true,    
    "transition_agent": false,    
    "loss_masking": false,       
    "enable_eos_bos_chars": true, 

    "batch_size": 4,       
    "eval_batch_size":2,   
    "r": 1,                 
    "wd": 0.000001,        
    "checkpoint": true,     
    "save_step": 1000,      
    "print_step": 100,      
    "tb_model_param_stats": true,     
    "batch_group_size": 8,  

    "run_eval": true,
    "test_delay_epochs": 2,       
    "min_seq_len": 0,       
    "max_seq_len": 150,     
    "output_path": "/Users/Gustavo/Desktop/TTS-main/tacotron2-ouputs",      
    "num_loader_workers": 4,        
    "num_val_loader_workers": 2,    
    "phoneme_cache_path": "/Users/Gustavo/Desktop/TTS-main",  
    "use_phonemes": true,           
    "phoneme_language": "es",     
    "text_cleaner": "phoneme_cleaners"
}
`
